solr/core/src/java/org/apache/solr/search/LegacyNumericRangeQueryBuilder.java

```diff
@@ -112,7 +113,7 @@ public Query getQuery(Element e) throws ParserException {
                 (lowerTerm == null ? null : Integer.valueOf(lowerTerm)),
                 (upperTerm == null ? null : Integer.valueOf(upperTerm)),
                 lowerInclusive,
-                upperInclusive);
+                upperInclusive, MultiTermQuery.CONSTANT_SCORE_REWRITE);
```

Comment:
ditto, re: "RewriteMethod in ctor"

See [comment here](https://github.com/apache/solr/pull/3053#discussion_r2050746215)